src/Utils/obiDialogParser.ts

```diff
@@ -214,28 +215,47 @@ export class ObiDialogParser {
 
     // Generates TrainDialog instances from the dialog tree.
     private async getTrainDialogs(node: ObiDialogNode): Promise<CLM.TrainDialog[]> {
-        return this.getTrainDialogsIter(node, [])
+        return this.getTrainDialogsIter(node, [], node.intent)
     }
 
     // Recursive helper.
-    private async getTrainDialogsIter(node: ObiDialogNode, currentRounds: CLM.TrainRound[]):
+    private async getTrainDialogsIter(node: ObiDialogNode, currentRounds: CLM.TrainRound[],
+        intent: string | undefined):
```

Comment:
nit: would either put all the arguments on separate lines or all same line, not mixed

Reply:
Done